db/migrate/20160412133816_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration
```

Comment:
I do want to share the same Category among many Categorizations, therefore I went with this.